palindrome_r.rb

```diff
@@ -0,0 +1,24 @@
+def palindrome_r(string)
+	if string.size == 0 || string.size == 1
+		return true
+	else
+		last_m_1 = string.size - 1
+		if string[0] == string[last_m_1]
```

Comment:
`string[string.size - 1]` is same as `string[-1]`. Your if statement can be modified to:
`if string[0] == string[-1]`